apps/vault/contracts/Vault.sol

```diff
@@ -1,118 +1,114 @@
-pragma solidity 0.4.18;
+pragma solidity 0.4.24;
 
-import "./VaultBase.sol"; // split made to avoid circular import
+import "@aragon/os/contracts/apps/AragonApp.sol";
 
-import "./connectors/ERC20Connector.sol";
-import "./connectors/ETHConnector.sol";
+import "@aragon/os/contracts/lib/token/ERC20.sol";
 
-import "@aragon/os/contracts/lib/misc/Migrations.sol";
 
+contract Vault is AragonApp {
```

Comment:
Maybe add `EtherTokenConstant` in the inheritance list, based on earlier discussion